src/curl.c

```diff
@@ -275,7 +276,7 @@ void
 curl_env_finish(struct curl_env *env)
 {
 	assert(env);
-	if (env->multi != NULL) {
```

Comment:
nit: I don't know that our codestyle says about it but IMP explicit comparison was better.

Reply:
Already fixed, thanks.